rails/app/assets/stylesheets/components/card.scss

```diff
@@ -146,11 +146,19 @@ $el-shadow: 0 1px 4px #bbb;
           }
         }
         p {
-          font-size: 0.7rem;
+          font-size: 0.85rem;
           line-height: 1.25;
           margin-top: 1rem;
           @include ui-font;
         }
+		.speakers {
+			img { 
+				width: 100px; 
```

Comment:
This is probably fine as a stop gap, but if someone uploads a non square image, this will stretch / squash (matter of perspective) rectangular images.

![image](https://user-images.githubusercontent.com/2660801/46384393-a66d5a80-c683-11e8-97c0-762b211ce74c.png)

Reply:
Very true. We've got all square images, so it will work for the immediate future (i.e. tomorrow), but maybe we can build this out more.